fxmanifest.lua

```diff
@@ -11,7 +11,14 @@ server_scripts {
     'server/main.lua'
 }
 
-client_script 'client/main.lua'
+client_script {
+	'@PolyZone/client.lua',
+    '@PolyZone/BoxZone.lua',
+    '@PolyZone/EntityZone.lua',
+    '@PolyZone/CircleZone.lua',
+    '@PolyZone/ComboZone.lua',
+	'client/main.lua'
```

Comment:
```suggestion
    'client/main.lua'
```
Incorrect linting / spacing